packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1253,6 +1253,40 @@ export default mixins(
 							this.nodeSelectedByName(node.name);
 						});
 					});
+
+					const tagsEnabled = this.$store.getters['settings/areTagsEnabled'];
+					if (tagsEnabled && Array.isArray(workflowData.tags)) {
+						const allTags: ITag[] = await this.$store.dispatch('tags/fetchAll');
+						const tagNames = new Set(allTags.map((tag) => tag.name));
```

Comment:
I might be missing something, but if we are deduplicating with a set at `addWorkflowTagIds`, do we need to deduplicate again when fetching? If not, here we could just have an array and use `includes()` instead of `has()` below.

Reply:
deduping this comment 😅